debian/control

```diff
@@ -0,0 +1,24 @@
+Source: libjsonv
+Priority: optional
+Maintainer: Remy Chibois <chybz@chybz.net>
+Build-Depends: debhelper (>= 9), cmake
+Standards-Version: 3.9.5
+Section: libs
+Homepage: https://github.com/tgockel/jsonv
+
+Package: libjsonv-dev
+Section: libdevel
+Architecture: any
+Depends: libjsonv1.0.0-pre0 (= ${binary:Version}), ${misc:Depends}
```

Comment:
Should this be one of those `configure_file` CMake-generated things? That way, you could just pick this control file up from `JSONV_VERSION`.